test/server/api.spec.js

```diff
@@ -1,46 +1,46 @@
 'use strict';
 
-var _ = require('underscore');
-var chai = require('chai');
-var chaiHttp = require('chai-http');
-var expect = chai.expect;
+const _ = require('underscore');
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const expect = chai.expect;
 chai.use(chaiHttp);
 
-var fakeCocktails = require('../mock/cocktails');
-var serverConfig = require('../../server/config');
-var utils = require('../../common/utils');
+const fakeCocktails = require('../mock/cocktails');
+const serverConfig = require('../../server/config');
+const utils = require('../../common/utils');
 require('../../server/index'); // Side effect: starts server when tests run
 
-var APP_PATH = 'http://localhost:' + serverConfig.port;
-var COCKTAIL_PATH = serverConfig.api.base + serverConfig.api.cocktailPath + '/';
-var fakeCocktail = fakeCocktails.fakeCocktail;
-var updatedFakeCocktail = fakeCocktails.updatedFakeCocktail;
+const APP_PATH = `http:\/\/localhost:${serverConfig.port}`;
```

Comment:
We don't _need_ to escape slashes inside template strings, but it doesn't hurt, and it keeps Sublime from thinking `//` is a comment.